tests/torch/test_compression_training.py

```diff
@@ -174,9 +174,7 @@ def finalize(self, dataset_dir, tmp_path_factory, weekly_models_path) -> "Compre
         return self
 
     def get_metric(self):
-        return self.sample_handler.get_metric_value_from_checkpoint(
```

Comment:
`tests.torch.test_compression_training.TestCompression._validate_train_metric` uses this function to get the *test* accuracy and compare it to the *ref* accuracy. Did you just make the whole test irrelevant so that it compares expected accuracy to expected accuracy?

Reply:
No, it's taking values from state dict of the checkpoint. I don't know why we need references both in code and in state dict, so I left only references in code

Reply:
The checkpoint's state dict *is* the test accuracy.

Reply:
https://github.com/openvinotoolkit/nncf/blob/8ceff7857b11b6019c5093c9a1ea32af10c4f15b/tests/torch/test_compression_training.py#L517-L522

Reply:
Please see https://github.com/openvinotoolkit/nncf/pull/2146